src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
It's better use enum  or literal types there, because you have clear requirements: https://prnt.sc/j5ad15TZAF1U
`enum Color {
  Red = 'red',
  Green = 'green',
  Blue = 'blue'
}`

Reply:
you are right, thanks